src/GUI/EFCorePowerTools/Handlers/ReverseEngineer/EfRevEngLauncher.cs

```diff
@@ -123,13 +125,23 @@ private bool IsDotnetInstalled()
             var startInfo = new ProcessStartInfo
             {
                 FileName = "dotnet",
-                Arguments = "--version",
+                Arguments = "--list-sdks",
             };
 
             var result = RunProcess(startInfo);
 
-            return result.StartsWith("3.", StringComparison.OrdinalIgnoreCase)
-                || result.StartsWith("5.", StringComparison.OrdinalIgnoreCase);
+            var isInstalled = false;
+            var sdks = result.Split(new string[] { "\r\n" }, StringSplitOptions.RemoveEmptyEntries);
+            foreach (var item in sdks)
+            {
+                isInstalled = item.StartsWith("3.1", StringComparison.OrdinalIgnoreCase);
```

Comment:
add `|| item.StartsWith("5."`

Reply:
https://github.com/ErikEJ/EFCorePowerTools/pull/696#discussion_r551754781

Reply:
Basically implement my initial comment: https://github.com/ErikEJ/EFCorePowerTools/pull/696#issuecomment-753895047

Reply:
Yes, but was that a test of your branch ?? - the error message shown indicates that it was not.

Reply:
Not my modified version, but the version downloaded from the market for testing

Reply:
Sorry to repeat my self: add `|| item.StartsWith("5."` and test your modififcations

Reply:
![image](https://user-images.githubusercontent.com/5714438/103624221-32ae0c80-4f74-11eb-9599-0b1ab68d46d1.png)